### DISCUSSION TITLE
I neeed help @NotLiam

### DISCUSSION THREAD

**Discussion by yPiuro** [anno-org/NotesRUs#91] [General]
@Notliam99 

![image](https://github.com/user-attachments/assets/8c855005-4243-4a95-ae90-547132e5bb1a)

## Shit is not working bludclat

**Comment by yPiuro** [comment]
Check snap aswell

**Reply by Notliam99** [reply]
bud the follow_id ect is a interger

**Comment by Notliam99** [comment]
@yPiuro You can auto increment char

**Reply by yPiuro** [reply]
huh?!

**Reply by Notliam99** [reply]
but probs use the uiid library to define them
